working/0516 - strict-raw-types/strict-raw-types.md

```diff
@@ -0,0 +1,125 @@
+# Strict raw types static analysis option
```

Comment:
As per comments in the strict inference review, consider moving this to resources/type-system?  Or alternatively add a resources/strict-checks/ directory?

Reply:
Done.